### PR TITLE
Integration test for the spark handler cloudconfig rendering

### PR DESCRIPTION
**Pull request by fiunchinho**

### CODE REVIEW

**Review by whites11 — APPROVED** [review]
LGTM

File: service/controller/resource/spark/testdata/simple_machinepool.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    - "1"
+    - "3"
+  replicas: 2
+  template:
+    metadata: { }
+    spec:
+      bootstrap:
+        configRef:
+          apiVersion: bootstrap.cluster.x-k8s.io/v1alpha3
+          kind: KubeadmConfig
```
Comment (tuommaki) [inline service/controller/resource/spark/testdata/simple_machinepool.yaml:25]: :question:

**Review by tuommaki — APPROVED** [review]
LGTM

File: service/controller/resource/spark/testdata/simple_machinepool.yaml (hunk 1 above)
Comment (fiunchinho) [reply service/controller/resource/spark/testdata/simple_machinepool.yaml:25]: I just copied it from somewhere else. I'll remove that part.

**Pull request merged**